.github/workflows/ci.yaml

```
@@ -23,3 +23,9 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - run: make docker-build
+
+  lint-chart:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - run: mkdir -p /tmp/bin && make lint-chart LOCALBIN=/tmp/bin
```

With 2c14df5, `LOCALBIN` environmental variable is no longer needed.

```suggestion
    - make lint-chart
```